swarm/pss/prox_test.go

```diff
@@ -241,7 +219,10 @@ func testProxNetwork(t *testing.T) {
 	}
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*120)
 	defer cancel()
-	snap := readSnapshot(t, nodeCount)
+	snap, err := simulation.ReadSnapshot(fmt.Sprintf("testdata/snapshot_%d.json", nodeCount))
```

Comment:
Nitpick; conformity - you don't have separate filename var here, like in https://github.com/ethersphere/go-ethereum/pull/1303/files#diff-fbced66e141c998cd0b7f816d30319a8R260

Reply:
fixed